src/Interface/Lists/RadarBotDirectory.xyz

```diff
@@ -0,0 +1,80 @@
+import { Service, ServicePostOptions } from '../Service'
+import { Util, IDResolvable } from '../../Utils/Util'
+import { Query } from '../../Utils/Constants'
+
+/**
+ * Represents the Radar Bot Directory service.
+ * @see https://docs.radarbotdirectory.xyz/
+ */
+export default class radarbotdirectory extends Service {
+  /** The values that can be used to select the service. */
+  static get aliases() {
+    return ['radar', 'radarbots', 'radarbotdirectory.xyz']
+  }
+
+  /** The logo URL. */
+  static get logoURL() {
+    return 'https://theoverlook.xyz/Images/RBD/botlistlogo.png'
+  }
+
+  /** Service's name. */
+  static get serviceName() {
+    return 'Radar Bot Directory'
+  }
+
+  /** The website URL. */
+  static get websiteURL() {
+    return 'https://radarbotdirectory.xyz'
+  }
+
+  /** The base URL of the service's API. */
+  static get baseURL() {
+    return 'https://radarbotdirectory.xyz/api/'
```

Comment:
```suggestion
    return 'https://radarbotdirectory.xyz/api'
```

This is just because of how we are concatenating URLs ([link](https://github.com/dbots-pkg/dbots.js/blob/1751a97c69f852e7df4b42d65e7b7a58d1ea928e/src/Interface/Service.ts#L103)), we should use a better way...